### DISCUSSION TITLE
Definitions for the LPeg library

### DISCUSSION THREAD

**Discussion by Josef-Friedrich** [LuaCATS/.github#1] [Transfers]
I created a repository containing the definitions for [LPeg](https://github.com/Josef-Friedrich/lpeg) (https://github.com/Josef-Friedrich/lpeg). I want to transfer this repo to LuaCATS.

**Comment by carsakiller** [comment]
Hello! Are you able to be a maintainer for lpeg once it is transferred or is it meant to be primarily community maintained?

**Comment by Josef-Friedrich** [comment]
I can maintain the LPeg lib

**Reply by carsakiller** [reply]
Okay. Just want to clarify, you don't have to transfer an addon to LuaCATS for it to be included in the addon manager. LuaCATS is meant to make it easier for the community to collaborate on addons together—especially those that can no longer be maintained by the original author.

If you still want to transfer it, though, head into the settings of your project and transfer it to me. I'll transfer it to LuaCATS and add you as a maintainer for LPeg 🙂

**Reply by Josef-Friedrich** [reply]
Thanks for the clarification. I would like to continue transferring the project and remain a maintainer. I have send you a transfer request.

**Reply by carsakiller** [reply]
It has been transferred to [LuaCATS/lpeg](https://github.com/LuaCATS/lpeg) and you are a maintainer.

If you would like to publish it to the addon manager, take a look at [LLS-Addons](https://github.com/LuaLS/LLS-Addons).
